crates/storage-api/src/partition_store/protobuf_types.rs

```diff
@@ -118,31 +123,35 @@ pub mod v1 {
     ));
 
     pub mod pb_conversion {
-        use crate::protobuf_types::ConversionError;
-        use crate::protobuf_types::v1::dedup_sequence_number::Variant;
-        use crate::protobuf_types::v1::enriched_entry_header::{
+        use crate::StorageError;
+        use crate::invocation_status_table::{CompletionRangeEpochMap, JournalMetadata};
+        use crate::partition_store::protobuf_types::ConversionError;
+        use crate::partition_store::protobuf_types::v1::dedup_sequence_number::Variant;
+        use crate::partition_store::protobuf_types::v1::enriched_entry_header::{
             AttachInvocation, Awakeable, BackgroundCall, CancelInvocation, ClearAllState,
             ClearState, CompleteAwakeable, CompletePromise, Custom, GetCallInvocationId,
             GetInvocationOutput, GetPromise, GetState, GetStateKeys, Input, Invoke, Output,
             PeekPromise, SetState, SideEffect, Sleep,
         };
-        use crate::protobuf_types::v1::entry::EntryType;
-        use crate::protobuf_types::v1::invocation_status::{
+        use crate::partition_store::protobuf_types::v1::entry::EntryType;
+        use crate::partition_store::protobuf_types::v1::invocation_status::{
             Completed, Free, Inboxed, Invoked, Suspended,
         };
-        use crate::protobuf_types::v1::invocation_status_v2::JournalTrimPoint;
-        use crate::protobuf_types::v1::journal_entry::completion_result::{
+        use crate::partition_store::protobuf_types::v1::invocation_status_v2::JournalTrimPoint;
```

Comment:
Maybe it's a good idea to clean up this. It's easier to read without this long prefix since it's a sub-module

```suggestion
        use invocation_status_v2::JournalTrimPoint;
```
This suggestion applies to all sub-modules here